### ISSUE TITLE
Allow editable dictionaries under /conf

### ISSUE THREAD

**Issue by bKalbitz** [orbinson/aem-dictionary-translator#46]
Hi,
first off, sorry I was not able to build the project on my own. Otherwise, I would provide a PR 😊
Not only /content also /conf is mutable content for AEMaaCS. We like to put our translation directly under /conf, to have a better access control and to hide the dictionaries from the Site Admin.
Unfortunately, the dictionaries under /conf are marked as not editable. First, I through it is an AC problem. But after looking into the code, I would say it is only this line of code:
https://github.com/orbinson/aem-dictionary-translator/blob/640effea45ee4568a46fc0e4bd2edac47f2f1248/core/src/main/java/be/orbinson/aem/dictionarytranslator/models/impl/DictionaryImpl.java#L71
Would it possible to allow also /conf there?
Thanks for your efforts and I appreciate the tool you have built here 😊

Kind regards,
Benjamin

**Comment by royteeuwen** [comment]
Hey @bKalbitz ,

Sure I'll see what I can do. 

Could you tell me why you weren't able to build the project on your own though? Do we need to update any readme? It should be a simple `mvn clean install`

**Comment by bKalbitz** [comment]
Hi, I've tested the change and it works. Thanks :)

Now the problem with my local build:
The build works now and I cannot reproduce it anymore because I switched my PC. But I think it was not related to the code base more to my locale setup.
If you are interested in unrelated details, my theory is that it was related to the bug JDK-8214129 that JDK 11 have problems with TLS 1.2 and some changes to our mirror repository, what then caused problems in downloading the dependencies from the mirror. I did not run in the error before, because my other dependencies were cached locally.
